### PR TITLE
feat(desktop): Added LocalStack desktop page and marked cockpit as deprecated

### PR DESCRIPTION
**Pull request by Pive01**
Do not merge until we decide to publish the desktop app

### CODE REVIEW

**Comment by lukqw** [comment]
@HarshCasper would be great to get your review in for deprecating the cockpit app and getting the desktop app featured on our docs!

This is also featured on our (soon-to-be) public repo: https://github.com/localstack/localstack-desktop
Would appreciate some feedback!

File: content/en/user-guide/tools/localstack-desktop/index.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+title: "LocalStack Desktop"
+weight: 8
+description: >
+  Getting started with LocalStack Desktop application
+tags: ["desktop"]
+aliases:
+  - /tools/localstack-desktop/
+  - /user-guide/tools/localstack-desktop/
```
Comment (HarshCasper) [inline content/en/user-guide/tools/localstack-desktop/index.md:9]: Please remove these

File: content/en/user-guide/tools/localstack-desktop/index.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+---
+title: "LocalStack Desktop"
+weight: 8
+description: >
+  Getting started with LocalStack Desktop application
```
Comment (HarshCasper) [inline content/en/user-guide/tools/localstack-desktop/index.md:5]:
```suggestion
  Getting started with the LocalStack Desktop application
```

File: content/en/user-guide/tools/localstack-desktop/index.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+tags: ["desktop"]
+aliases:
+  - /tools/localstack-desktop/
+  - /user-guide/tools/localstack-desktop/
+---
+
+LocalStack Desktop is a desktop client that allows users to easily control and interact their LocalStack instance. With LocalStack Desktop, users can start and stop their LocalStack instance with a single click, create a new container, view logs, interact with LocalStack container via cli and use our resource browser.
```
Comment (HarshCasper) [inline content/en/user-guide/tools/localstack-desktop/index.md:12]: Is it only available to Pro/Team users? Or generally available?

**Review by HarshCasper — CHANGES_REQUESTED** [review]

File: content/en/user-guide/tools/localstack-desktop/index.md (hunk 3 above)
Comment (Pive01) [reply content/en/user-guide/tools/localstack-desktop/index.md:12]: Is available for everybody but you need to log in to use the resource browser

**Review by lukqw — APPROVED** [review]

**Review by HarshCasper — APPROVED** [review]

File: content/en/user-guide/tools/localstack-desktop/index.md [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+<img src="localstack-desktop-containers.png" alt="LocalStack Desktop" title="LocalStack Desktop" width="900" />
+
+## Installation
+
+You can install LocalStack Desktop by downloading the latest executables for your platform from the [web application](https://app.localstack.cloud/download) or by installing it directly from the [Microsoft Store](https://www.microsoft.com/store/apps/9NTRNFT9ZWS2).
+To install LocalStack Desktop, **Docker** is the only prerequisite. 
+
+## Features
+
+LocalStack Desktop helps users to interact with their LocalStack instance with a simple and intuitive UI. Some of the features of LocalStack Desktop includes the ability to : Control LocalStack, Interact with LocalStack, get LocalStack insights and use the Resource browser.
```
Comment (MarcelStranak) [inline content/en/user-guide/tools/localstack-desktop/index.md:19]:
```suggestion
LocalStack Desktop helps users to interact with their LocalStack instance with a simple and intuitive UI. Some of the features of LocalStack Desktop includes the ability to: Control LocalStack, Interact with LocalStack, get LocalStack insights and use the Resource browser.
```

File: content/en/user-guide/tools/localstack-desktop/index.md [hunk 5]
```diff
@@ -5,13 +5,13 @@ description: >
...
 ---
 
-LocalStack Desktop is a desktop client that allows users to easily control and interact their LocalStack instance. With LocalStack Desktop, users can start and stop their LocalStack instance with a single click, create a new container, view logs, interact with LocalStack container via cli and use our resource browser.
+LocalStack Desktop is a desktop client that allows users to easily control and interact with their LocalStack instance. Using LocalStack Desktop, users can start and stop their LocalStack instance with a single click, create a new container, view logs, interact with LocalStack container via cli and use our resource browser.
 
 <img src="localstack-desktop-containers.png" alt="LocalStack Desktop" title="LocalStack Desktop" width="900" />
 
 ## Installation
 
-You can install LocalStack Desktop by downloading the latest executables for your platform from the [web application](https://app.localstack.cloud/download) or by installing it directly from the [Microsoft Store](https://www.microsoft.com/store/apps/9NTRNFT9ZWS2).
```
Comment (Pive01) [inline content/en/user-guide/tools/localstack-desktop/index.md]: We want all the download to go through the web app

**Pull request merged**